roles/network_plugin/calico/tasks/check.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: "Check supported calico versions"
+  assert:
+    that:
+     - "calico_version in calico_crds_archive_checksums.keys()"
```

Comment:
Here we need to fix indent to pass yamlint job.
```
$ yamllint --strict .
./roles/network_plugin/calico/tasks/check.yml
  5:6       error    wrong indentation: expected 6 but found 5  (indentation)
```

Reply:
Sorry about that, it is fixed in the last push.